Adding Golang Cafe (youtube) & Libraries (new section)

pitakill opened the pull request:
1. Why is this change necessary?
More resources, more knowledge to share

2. How does it address the issue?
Add the new resources to the tracker of them

3. What side effects does this change have?
Gain more knowledge :-)

pitakill committed:
Adding Golang Cafe (youtube) & Libraries (new section)

1. Why is this change necessary?
More resources, more knowledge to share

2. How does it address the issue?
Add the new resources to the tracker of them

3. What side effects does this change have?
Gain more knowledge :-)

danvergara commented on README.md:
```diff
@@ -20,6 +20,13 @@
 | [Go Secure Coding Practices](https://checkmarx.gitbooks.io/go-scp) | Web Application Secure Coding Practices es una guía para cualquiera que esté usando el lenguaje de programación Go y tenga el objetivo de usarlo para desarrollo web | Inglés |
 | [Spaceship Go (stdlib journey)](https://blasrodri.github.io/spaceship-go-gh-pages) | Este es un conjuto de notas de mi viaje en una parte de la librería estándar de Go | Inglés |
 
+## Librerías
```
Muchas gracias por tu contribución.
Un detalle aquí. ¿Podemos usar Bibliotecas en lugar de Librerías? Hasta donde sé es la traducción correcta.

pitakill replied:
Ahhh sí, justo eso me vino a la mente. Hay debate al respecto y no lo tengo claro. xD

Pero le pongo Bibliotecas :-)

pitakill committed:
Update the translation of Library to Biblioteca

1. Why is this change necessary?
There is debate about the correct translation of Library. We think the
correct one is Bibloteca in spanish

2. How does it address the issue?
Change the words accordingly

3. What side effects does this change have?
Better Spanish :-)
